### PR TITLE
Update page in Manage your Asset Catalog section “tags_attributes_categories”

### PR DESCRIPTION
**Pull request by robertengaxway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by lbadenhop** [comment]
@robertengaxway I fixed the markdown error and reviewed/edited the content. Is this topic complete? Are you going to document "How to" apply tags, attributes and categories

**Review by cbordier — APPROVED** [review]
Looks good to me

File: content/en/docs/manage_asset_catalog/tags_attributes_categories.md [hunk 1]
```diff
@@ -3,5 +3,47 @@ title: Tags, Attributes and Categories
 linkTitle: Tags, Attributes and Categories
 weight: 40
 ---
+Use tags, attributes, and categories to edit the details of your assets. This can make the curation and organization of assets more valuable.
 
-Content coming soon.
+## Add or remove tags
+
+Tags are keywords that can assigned to catalog assets and used to search for assets. You can create tags with any special characters. For example, `_ - & () \ []` and all types of special language characters can be used to create tags.
```
Comment (georgiana-caras) [inline content/en/docs/manage_asset_catalog/tags_attributes_categories.md:10]: be assigned

File: content/en/docs/manage_asset_catalog/tags_attributes_categories.md [hunk 2]
```diff
@@ -3,5 +3,47 @@ title: Tags, Attributes and Categories
...
+
+1. Select an asset in the *Asset Catalog*.
+2. Either select a asset version in the Draft state or click either **Create Draft** or **Edit Asset** on the right-side details.
```
Comment (georgiana-caras) [inline content/en/docs/manage_asset_catalog/tags_attributes_categories.md:13]: an asset

File: content/en/docs/manage_asset_catalog/tags_attributes_categories.md [hunk 3]
```diff
@@ -3,5 +3,47 @@ title: Tags, Attributes and Categories
...
+
+    * If a draft version was selected, then in the right-side details panel, click on the Attributes field and enter your new attribute and value to be added. To remove an attribute, click the **X** next to the attribute key=value.
+    * If you selected **Create Draft** or **Edit Asset**, the wizard will appear and click **Categories, Tags, & Attributes**. Then click on the Attribute field and enter your new attribute Key and Value to be added.  To remove a category, click the **X** next to the attribute key=value.
+
+3. Press **Enter** or click **+** to add the new attribute you have typed to the list of attributes.
+4. Click **Save** or **Save Draft & Exit**.
+
+## Manage categories
+
+Searching through an unordered list of assets can become a dauting tasks and make assets less organized. Central Admin and Product Manager users can create categories to group assets in the Amplify Asset Catalog to make searching for assets more effective and more organized. Examples of categories are `Finance` or `Medicine`.
```
Comment (georgiana-caras) [inline content/en/docs/manage_asset_catalog/tags_attributes_categories.md:36]: remove product manager as they cannot create

File: content/en/docs/manage_asset_catalog/tags_attributes_categories.md [hunk 4]
```diff
@@ -3,5 +3,47 @@ title: Tags, Attributes and Categories
...
+
+Categories allow for a broader grouping of assets into different topics or themes. They are created once, and multiple assets can be assigned to them. One asset can be in multiple categories.
+
+Only users that are assigned the **Central Admin** or **Product Manager** role can create, edit and delete categories. Categories are created at the Organization level and available to all teams to apply them to their assets.
```
Comment (georgiana-caras) [inline content/en/docs/manage_asset_catalog/tags_attributes_categories.md:40]: catalog manager cannot create categories, please remove

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

File: content/en/docs/manage_asset_catalog/tags_attributes_categories.md (hunk 2 above)
Comment (robertengaxway) [reply content/en/docs/manage_asset_catalog/tags_attributes_categories.md:13]: > an asset

edited

**Review by robertengaxway — COMMENTED** [review]
I updated the changes suggested by Georgi

**Review by georgiana-caras — APPROVED** [review]

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**